tsl/test/expected/move.out

```diff
@@ -0,0 +1,469 @@
+                   length(T1.value), length(T2.value)
+            FROM T1 FULL OUTER JOIN T2 ON T1 = T2
+            WHERE T1 IS NULL OR T2 IS NULL
+        
+ time | time | temp | temp | location | location | substring | substring | length | length 
+------+------+------+------+----------+----------+-----------+-----------+--------+--------
+(0 rows)
+
+SELECT * FROM ensure_scans_work('cluster_test', should_output => 'expected', bitmapscan => true) \gexec
+
+            SET LOCAL enable_seqscan=f;
+            SET LOCAL enable_indexscan=f;
+            SET LOCAL enable_bitmapscan=t;
+            EXPLAIN (costs off) SELECT * FROM cluster_test WHERE time=7
+        
+                               QUERY PLAN                                
+-------------------------------------------------------------------------
+ Append
+   ->  Bitmap Heap Scan on _hyper_1_2_chunk
+         Recheck Cond: ("time" = 7)
+         ->  Bitmap Index Scan on _hyper_1_2_chunk_cluster_test_time_idx
+               Index Cond: ("time" = 7)
+(5 rows)
+
+
+            WITH T1 as (SELECT * FROM cluster_test WHERE time=7),
+                 T2 as (SELECT * FROM expected WHERE time=7)
+            SELECT T1.time, T2.time, T1.temp, T2.temp, T1.location, T2.location,
+                   substring(T1.value for 30), substring(T2.value for 30),
+                   length(T1.value), length(T2.value)
+            FROM T1 FULL OUTER JOIN T2 ON T1 = T2
+            WHERE T1 IS NULL OR T2 IS NULL
+        
+ time | time | temp | temp | location | location | substring | substring | length | length 
+------+------+------+------+----------+----------+-----------+-----------+--------+--------
+(0 rows)
+
+COMMIT;
+-- Show chunk indexes
+SELECT * FROM test.show_indexes('_timescaledb_internal._hyper_1_1_chunk');
+                              Index                               |  Columns   | Expr | Unique | Primary | Exclusion | Tablespace 
+------------------------------------------------------------------+------------+------+--------+---------+-----------+------------
+ _timescaledb_internal._hyper_1_1_chunk_cluster_test_location_idx | {location} |      | f      | f       | f         | 
+ _timescaledb_internal._hyper_1_1_chunk_cluster_test_time_idx     | {time}     |      | f      | f       | f         | 
+(2 rows)
+
+SELECT * FROM test.show_indexes('_timescaledb_internal._hyper_1_2_chunk');
+                              Index                               |  Columns   | Expr | Unique | Primary | Exclusion | Tablespace 
+------------------------------------------------------------------+------------+------+--------+---------+-----------+------------
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_location_idx | {location} |      | f      | f       | f         | 
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx     | {time}     |      | f      | f       | f         | 
+(2 rows)
+
+\set ON_ERROR_STOP 0
+-- cannot move a chunk with no reorder index on first call 
```

Comment:
maybe add a test for what happens if the previous reorder index is dropped?

Reply:
added in reorder tests